services/galley/test/integration/API/Util.hs

```diff
@@ -997,45 +1018,44 @@ listRemoteConvs remoteDomain uid = do
   pure $ filter (\qcnv -> qDomain qcnv == remoteDomain) allConvs
 
 postQualifiedMembers ::
-  (HasGalley m, MonadIO m, MonadHttp m) =>
+  (MonadReader TestSetup m, MonadIO m, MonadHttp m) =>
   UserId ->
   NonEmpty (Qualified UserId) ->
   ConvId ->
   m ResponseLBS
 postQualifiedMembers zusr invitees conv = do
-  g <- viewGalley
+  g <- view tsUnversionedGalley
   let invite = InviteQualified invitees roleNameWireAdmin
   post $
     g
-      . paths ["conversations", toByteString' conv, "members", "v2"]
+      . paths ["v1", "conversations", toByteString' conv, "members", "v2"]
```

Comment:
unversioned + v1 with the v2 suffix? This is a rather strange combination.

Reply:
Right, that's why we're removing those v2 suffixes everywhere.